README.md

```diff
@@ -1 +0,0 @@
-# nimble-ansible-modules
```

Comment:
We still need a README in the root of the repo to point out licensing, contributing etc.

Reply:
ok will add later..

Reply:
as far as license goes we can copy the same license which we have in another readme file.

for contribution,. is the below link ok?
https://hpe-storage.github.io/nimble-python-sdk/legal/contributing/index.html

Reply:
i have taken the contibution.md from container repo .. and will put it in our repo for ansible

Reply:
> I feel we are doing way too much in this one PR. Can we limit the scope and move the galaxy changes into a separate PR? Maybe not now but in the future.

sure george.. will move all new release related stuffs like README, galaxy.yml license, etc to a separate PR.